### PR TITLE
Update images and related information of using_area_2d

### PR DESCRIPTION
**Pull request by Piralein**
- Updated images
- Updated information related to existing properties and their order.

### CODE REVIEW

File: tutorials/physics/using_area_2d.rst [hunk 1]
```diff
@@ -28,21 +28,21 @@ Area properties
...
-
-The "Audio Bus" section allows you to override audio in the area, for example to
+The ``Audio Bus`` section allows you to override audio in the area, for example to
 apply an audio effect when the player moves through.
 
 Note that Area2D extends :ref:`CollisionObject2D <class_CollisionObject2D>`, so it
-also provides properties inherited from that class, such as ``input_pickable``.
+also provides properties inherited from that class.
+The ``Collision`` section of ``CollisionObject2D`` is where you configure the
+area's collision layer(s)and mask(s).
```
Comment (AThousandShips) [inline tutorials/physics/using_area_2d.rst:45]:
```suggestion
area's collision layer(s) and mask(s).
```

File: tutorials/physics/using_area_2d.rst [hunk 2]
```diff
@@ -28,21 +28,21 @@ Area properties
 
 Areas have many properties you can use to customize their behavior.
 
-.. image:: img/area2d_properties.png
+.. image:: img/area2d_properties.webp
 
-The first eight properties are used to configure the area's physics override
-behavior. We'll look at how to use those in the section below.
+The ``Gravity``, ``Linear Damp`` and ``Angular Damp`` sections
```
Comment (AThousandShips) [inline tutorials/physics/using_area_2d.rst:33]:
```suggestion
The ``Gravity``, ``Linear Damp``, and ``Angular Damp`` sections
```
As per [guidelines](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html)

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you! Merged.

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.1

**Pull request merged**